packages/db-states-redis/src/lib/states/statesInRedisClient.ts

```diff
@@ -20,32 +20,16 @@ type JSONDecoderValue = Record<string, any>;
 /**
  * Decodes a JSON with buffer value
  *
- * @param key
+ * @param _key
  * @param value
  */
-function bufferJsonDecoder(key: string, value: JSONDecoderValue): Buffer | JSONDecoderValue {
+function bufferJsonDecoder(_key: string, value: JSONDecoderValue): Buffer | JSONDecoderValue {
```

Comment:
maybe we could get rid of this param if it is unused as it is just an internal method this wouldn't be a problem

Reply:
I mean in this function we pass it but we don't need it
<img width="981" alt="Bildschirmfoto 2024-11-15 um 13 06 21" src="https://github.com/user-attachments/assets/5941434d-a2b6-4571-b929-301af8ed41d2">

Reply:
Ah sorry, now I see. you are right!